### ISSUE TITLE
import-module : SafeCommands entry for uname does not hold a reference to the proper command. - macos

### ISSUE THREAD

**Issue by DylanGraham** [pester/Pester#1113]
## 1. General summary of the issue

After installation on mac os, importing the module fails.
`Install-Module -Name Pester -Force -SkipPublisherCheck`

## 2. Describe Your Environment

```
Pester version     : 4.4.2 /usr/local/share/powershell/Modules/Pester/4.4.2/Pester.psd1
PowerShell version : 6.1.0
OS version         : Unix 17.7.0.0
```

## 3. Expected Behavior

Import succeeds.

## 4.Current Behavior
```
> Import-Module pester -Verbose
VERBOSE: Loading module from path '/usr/local/share/powershell/Modules/Pester/4.4.2/Pester.psd1'.
VERBOSE: Populating RepositorySourceLocation property for module Pester.
VERBOSE: Loading module from path '/usr/local/share/powershell/Modules/Pester/4.4.2/Pester.psm1'.
Import-Module : SafeCommands entry for uname does not hold a reference to the proper command.
At line:1 char:1
+ Import-Module pester -Verbose
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : OperationStopped: (SafeCommands en...proper command.:String) [Import-Module], RuntimeException
+ FullyQualifiedErrorId : SafeCommands entry for uname does not hold a reference to the proper command.,Microsoft.PowerShell.Commands.ImportModuleCommand
```

**Comment by DylanGraham** [comment]
Hi @nohwnd 
I have a 2018 and 2017 macbook pro here, but both are on High Sierra 10.13.6.
I just tried on the second laptop, which had powershell 6.0.4 and have the same error. I then upgraded to powershell 6.1.0 with no change to Pester.
Is powershell installed from `brew cask` sufficient?
Thanks

**Comment by DylanGraham** [comment]
I now updated to 10.14 Mojave and have the same issue

**Comment by nohwnd** [comment]
@DylanGraham yes, this is a new feature in the module, I still cannot reproduce it. :/ @JamesWTruher did you encounter this issue?

**Comment by TheMacStack** [comment]
I hit something similar with 4.4.2 on a Linux centos docker image, downgrading Pester to 4.4.1 solved it

dockerfile I was using if you want to try and recreate it:
```
FROM mcr.microsoft.com/powershell:6.1.0-rc.1-centos-7
RUN pwsh -c "Install-Module -Name Az -Force"
RUN pwsh -c "Install-Module -Name Pester -Force"
```

error I was getting when running in the above image in some ci/cd:
```
$ pwsh -c Import-Module Pester
Import-Module : SafeCommands entry for id does not hold a reference to the proper command.
At line:1 char:1
+ Import-Module Pester
+ ~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : OperationStopped: (SafeCommands en...proper command.:String) [Import-Module], RuntimeException
+ FullyQualifiedErrorId : SafeCommands entry for id does not hold a reference to the proper command.,Microsoft.PowerShell.Commands.ImportModuleCommand
```

**Comment by nohwnd** [comment]
@TheMacStack thanks will try, hopefully this is easy to reproduce this way.

**Comment by nohwnd** [comment]
Okay this was fixed by #1126 the problem here is that there are multiple instances of `id` and `uname` command and so the sanity check fails. Will release as part of 4.4.3 during weekend.

```
CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Application     id                                                 0.0.0.0    /usr/bin/id
Application     id                                                 0.0.0.0    /bin/id
```
